Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionTreeViewItems.cs

```diff
@@ -239,8 +239,21 @@ public ActionTreeItem(SerializedProperty actionMapProperty, SerializedProperty a
         public override GUIStyle colorTagStyle => Styles.greenRect;
         public bool isSingletonAction => actionMapProperty == null;
 
-        public override string expectedControlLayout =>
-            property.FindPropertyRelative("m_ExpectedControlType").stringValue;
+        public override string expectedControlLayout
+        {
+            get
+            {
+                var expectedControlType = property.FindPropertyRelative("m_ExpectedControlType").stringValue;
+                if (!string.IsNullOrEmpty(expectedControlType))
+                    return expectedControlType;
+
+                var type = property.FindPropertyRelative("m_Type").intValue;
+                if (type == (int)InputActionType.Button)
+                    return "Button";
+
+                return null;
+            }
+        }
```

Comment:
Unrelated change. Tweak to a previous PR. Just couldn't be bothered to make this a separate PR...